src/en/howto-machineupdateseries.md

```diff
@@ -0,0 +1,137 @@
+Title: Change a machine's series pre-juju2.3
+
+#  Change a machine's series pre-juju2.3
+
+## Overview
+
+If using juju 2.3 or later, you can run the `juju update-series` command.
+
+The `juju update-series <machine> <series>` will update the series saved
+for the given machine, as well as any unit's on that machine.  When a
+unit's series is updated, the config-changed hook will be triggered.
+
+This guide will cover how to do this prior to juju 2.3
+
+!!! Note: charms may have to be updated to take advantage series info change.
+
+## Prerequisites
+
+1. Update the series on the machine
+
+2. Check that the current revision of the charm(s) in use for on the machine
+supports the series you wish to use in the future.  This checked on in the juju
+charm store.
+
+3. Name of the machine.
+
+4. Name of the model where the machine lives.
+
+5. Admin privilages to the controller where the model lives.
+
+## 
+
+Start by switching to the juju controller where the changes will happen.
+
+```
+$ juju switch localhost-test:admin/default
+```
```

Comment:
We use syntax highlighting using a "type" with 'bash', 'yaml', 'json', and 'no-highlight' being the most common. We also do not include the prompt.

>\```bash
>juju switch localhost-test:admin/default
>\```

We also separate input from output. Typically the output (which uses the 'no-highlight' type) is prepended by something like:

>This will give output closely resembling:

Try being creative with the above. It gets tiresome really quickly by using the same everywhere. An option is to sometimes include the command (if it's simple like `juju status` for example) within the preceding text itself (e.g. "Output to the `juju status` command will look like:").

Reply:
Added the highlighting types.  Tried to make the output more interesting?